### DISCUSSION TITLE
Error creating admin cert while using Dockerfile?

### DISCUSSION THREAD

**Discussion by SleepyKiiyu** [dogtagpki/pki#4189] [Q&A]
Hello everyone! 

Im trying to use the docker image of dogtag to configure the PKI but im having an error while running docker-compose up (image of docker-compose file attached below). Error appears when creating .crt of admin user (added photo below with exact error), does anyone know how to solve this issue?

Error log: 
![image](https://user-images.githubusercontent.com/23573093/195134220-9845474c-425c-4ae4-b62b-2715401080ea.png)

Docker-compose file:
![image](https://user-images.githubusercontent.com/23573093/195137034-81acbc83-da8a-43ad-bf8b-e68879a83990.png)

Thanks a lot!

**Comment by edewata** [comment]
Hi, the CA container is still under development. This is the only method that currently works:
* https://github.com/dogtagpki/pki/wiki/Deploying-CA-Container
* https://github.com/dogtagpki/pki/blob/master/.github/workflows/ca-container-test.yml
